core/src/ops/quant.rs

```diff
@@ -146,6 +147,119 @@ fn info_quantize_linear_i8(q: &QuantizeLinearI8) -> TractResult<Vec<String>> {
     )])
 }
 
+fn dynamic_quantize_linear_f32_u8(x: f32, scale: f32, zero_point: u8) -> u8 {
+    (((x / scale).round() as i32) + zero_point as i32)
+        .max(u8::min_value() as i32)
+        .min(u8::max_value() as i32) as u8
```

Comment:
why not simply hard code this?
```suggestion
        .max(0.)
        .min(255.) as u8
```

Reply:
I don't like magic numbers in the code. `u8::min_value()` is more explicit. We could use `u8::MIN` but the rest of the code is using `min_value`.

Reply:
ok, makes sense.